recipes/gradle/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "gradle" %}
+{% set version = "5.3.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gradle/gradle/archive/v5.3.1.tar.gz
+  sha256: f5bc12b30a44ffe79295c5a9a7ac6b1cfa52e99226cb40e6d18a05af91c73ea4
+  fn: gradle-{{ version }}.tar.gz
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+requirements:
+  host:
+    - coreutils >=8
+    - openjdk >=11
+  run:
+    - openjdk >=11
+
+test:
+  commands:
+    - gradle -v
+
+about:
+  home: https://gradle.org
+  license: Apache 2.0
+  license_file: LICENSE.txt
```

Comment:
```suggestion
  license_file: LICENSE
```

Reply:
> license_file entry is missing, but is expected.
Should I leave the "license_file: LICENSE" entry even if the file is not included in the recipe ?

Reply:
Yes, it’ll get it from the tarball, which is generally preferred to ensure it’s kept up to date.